src/config/side-panel-tabs/admin-tabs-by-stage.ts

```diff
@@ -49,10 +49,15 @@ export const ADMIN_TABS_BY_STAGE: Record<Stage, TabType[]> = {
     PAGES.preferenceSubmissions,
   ],
   // TODO: fill these in
-  [Stage.READER_BIDDING]: [PAGES.allReaders, PAGES.addReaders],
+  [Stage.READER_BIDDING]: [
+    PAGES.allReaders,
+    PAGES.addReaders,
+    PAGES.readerPreferenceOverview,
+  ],
   [Stage.READER_ALLOCATION]: [
     PAGES.allReaders,
     PAGES.addReaders,
```

Comment:
Since we don't block (I assume) readers from submitting preferences during this stage (to allow late submissions), maybe we should leave the page here for admins to be able to inspect who hasn't done their work. Thoughts?

```suggestion
    PAGES.addReaders,
    PAGES.readerPreferenceOverview
```

Reply:
good idea